src/WellIO.jl

```diff
@@ -1,5 +1,10 @@
 module WellIO
 
-# Write your package code here.
+using FileIO
 
-end
+include("readFile.jl")
+
+function load(file::File{format"LAS"})
+    f = readFile(file)
```

Comment:
You are missing an `end`. This module is not even compiling, is it? Can you please put more effort to load the package in a session and start by writing some tests?

Reply:
oops. Must have deleted by mistake!